pal/src/map/common.cpp

```diff
@@ -49,7 +49,7 @@ INT W32toUnixAccessControl( IN DWORD flProtect )
         MemAccessControl = PROT_EXEC | PROT_READ | PROT_WRITE;
         break;
     case PAGE_EXECUTE :
-        MemAccessControl = PROT_EXEC;
+        MemAccessControl = PROT_EXEC | PROT_READ; // WinAPI PAGE_EXECUTE also implies readable
```

Comment:
This boils down to that if we want PAL implementation carries the same meaning as Windows or not. Since  on Windows : https://msdn.microsoft.com/en-us/library/windows/desktop/aa366786(v=vs.85).aspx, PAGE_EXECUTE on Windows implies readable, only writes will generate AV. There is no such thing on Windows that PAGE_EXECUTE means execute only, not readable. This PR is to ensure that on PAL implementation emulates the Windows API.